waterlog/src/Components/AlertTable.js

```diff
@@ -1,66 +1,39 @@
 import React, { Component } from 'react';
-import { LowStatusIcon, MediumStatusIcon, HighStatusIcon } from './AlertBox';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
-import { fetchSegmentsLeaks } from '../actions/SegmentLeaksActions';
+import { fetchAlerts } from '../actions/AlertsAction';
 import Loader from './Loader';
-
-export const getStatusIcon = function (severity) {
-	switch (severity.toLowerCase()) {
-		case 'high':
-			return HighStatusIcon();
-		case 'low':
-			return LowStatusIcon();
-		case 'medium':
-			return MediumStatusIcon();
-		default:
-			return null;
-	}
-};
-const formatDate = (date) => {
-    var d = new Date(date),
-        month = '' + (d.getMonth() + 1),
-        day = '' + d.getDate(),
-        year = d.getFullYear();
-
-    if (month.length < 2) month = '0' + month;
-    if (day.length < 2) day = '0' + day;
-
-    return [month, day, year].join('/');
-}
+import { formatDate, getStatusIcon } from './../utils'
```

Comment:
add a semi-colon at the end of this import

Reply:
Fixed